<launch>

  <node pkg="floor_plane_ransac" type="floor_plane_ransac" name="floor_plane_ransac" output="screen"
      launch-prefix="xterm -e gdb --args">
      <param name="base_frame" value="/bubbleRob" />
      <param name="max_range" value="2.0" />
      <param name="n_samples" value="15000" />
      <param name="tolerance" value="0.01" />

      <remap from="/floor_plane_ransac/scans" to="/vrep/depthSensor"/>
  </node>
</launch>
